applications/nrf5340_audio/Kconfig

```diff
@@ -33,6 +33,24 @@ config TRANSPORT_CIS
 endchoice
 
 #----------------------------------------------------------------------------#
+if (TRANSPORT_BIS && AUDIO_DEV = 1)
+rsource "samples/broadcast_sink/Kconfig.defaults"
+endif # TRANSPORT_BIS && AUDIO_DEV = 1
```

Comment:
Feature creep 1: Back in the day, I recall that defining this 1 as "HEADSET" posed some issues. Perhaps there is better support for this now to improve readability?

Reply:
The problem is that we define HEADSET=1 in the CMakeLists.txt and that is included later than the Kconfigs